src/process/encryptExample.ts

```diff
@@ -0,0 +1,15 @@
+// Example with configuration
+import CryptoJS from "crypto-js";
+import createProcess from "../process";
+
+export default (secret: string) => {
```

Comment:
This filename suggests it's an example, but the other files in this dir don't have "example" in the name.  Is there a significance to the naming conventions?

Reply:
I've remove the "example" from the name, but in the main export I've called it "symetricEncrypt" to distinguish it from something you'd want to use with your private key

Reply:
Nice!  Sounds good to me